packages/manager/modules/netapp/src/dashboard/routing.js

```diff
@@ -26,6 +26,9 @@ export default /* @ngInject */ ($stateProvider) => {
         trackClick('create-volume');
         return $state.go('netapp.dashboard.volumes.create');
       },
+      goToDashboard: /* @ngInject */ ($state) => () => {
+        return $state.reload();
+      },
```

Comment:
```suggestion
      goToDashboard: /* @ngInject */ ($state) => () => $state.reload(),
```
This does not seem to be used, did you intend to use it in packages/manager/modules/netapp/src/dashboard/index/controller.js instead of window.location.reload() ?